### PR TITLE
Sort spec list by number

### PR DESCRIPTION
**Pull request by jarrodmillman**
Depends on https://github.com/scientific-python/specs/pull/306

### CODE REVIEW

File: layouts/shortcodes/page_gallery.html [hunk 1]
```diff
@@ -1,21 +1,8 @@
...
-        </div>
-        {{ .Title }}
-      </a>
-    </div>
-  {{ end }}
-</div>
+{{ $items := slice }}
+{{ $section := .Page.GetPage (.Get "pages") }}
+{{ range (or $section.Pages ($section.Resources.ByType "page")) }}
+  {{ $body := printf "{{< image >}}\n src = '%s'\n alt = '%s Logo'\n {{< /image >}}" .Params.avatar .Title }}
```
Comment (stefanv) [inline layouts/shortcodes/page_gallery.html:5]: Does the image shortcode support `loading="lazy"`?

File: layouts/shortcodes/page_gallery.html [hunk 2]
```diff
@@ -1,21 +1,8 @@
...
+  {{ $items = $items | append (dict "body" $body "header" .Title "link" .Params.repository "classcard" "text-center" "type" "card") }}
+{{ end }}
+{{- partial "_elements/grid.html" (merge .Params (dict "items" $items)) -}}
```
[the remark this answers is not in the document]
Comment (jarrodmillman) [reply layouts/shortcodes/page_gallery.html:8]: It has the optional parameters (outline and columns).

File: layouts/shortcodes/page_gallery.html (hunk 1 above)
Comment (jarrodmillman) [reply layouts/shortcodes/page_gallery.html:5]: I can check, but we can easily add it.

Comment (jarrodmillman) [reply layouts/shortcodes/page_gallery.html]: See #587

File: layouts/shortcodes/page_gallery.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (jarrodmillman) [reply layouts/shortcodes/page_gallery.html]: See #587

**Pull request merged**